integration_test/util/restartableSyncService.js

```diff
@@ -6,7 +6,7 @@ class RestartableSyncService extends DataSyncService {
     this.app.server = stoppable(this.app.server, 0)
     await this.app.server.listen(this.port)
     this.log.info(`Server is now running on http://localhost:${this.port}`)
-    await new Promise(resolve => setTimeout(resolve, 1000))
+    await new Promise(resolve => setTimeout(resolve, 200))
```

Comment:
IMHO in order to follow the best practices and to avoid magic numbers the `200` here should be a const with a name which clarifies its propose.

Reply:
@camilamacedo86 Thanks for this. This is actually great point. This is in order to avoid a timing bug (for some reason the thing isn't always initialized instantly).

I've been working on this PR for the last couple of days and I forgot to come back and tidy this up :)